File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+---
+title: Convert Markdown to Word document in C# | DocIO | Syncfusion
+description: Convert Markdown to Word document in C# using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Markdown to Word Conversion
+
+Markdown is a lightweight markup language that adds formatting elements to plain text documents. The .NET Word (DocIO) library supports to convert Markdown to Word document and vice versa, which mostly follows the CommonMark specification and GitHub-flavored syntax.
+
+## Convert Markdown to Word document
+
+Convert an existing markdown file to Word document (DOC, DOCX and RTF) using the .NET Word (DocIO) library.
+
+The following code example shows how to convert Markdown to Word document.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+//Open an existing Markdown file.
+using (WordDocument document = new WordDocument("Input.md", FormatType.Markdown))
+{
+    //Save as a Word document.
+    document.Save("MarkdownToWord.docx", FormatType.Docx);
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Open an existing Markdown file.
+Using document As WordDocument = New WordDocument("Input.md", FormatType.Markdown)
+    'Save as a Word document.
+    document.Save("MarkdownToWord.docx", FormatType.Docx)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Open the file as a Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.md"))
+{
+    //Load the file stream into a Markdown file.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Markdown))
+    {
+        //Save as a Word document into the MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        await document.SaveAsync(stream, FormatType.Docx);
+        //Save the stream as a Word document in the local machine.
+        Save(stream, "MarkdownToWord.docx");
+    }
+//Please refer to the following link to save the Word document in UWP platform
```

Comment:
//Please refer to the following link to save a Word document in the UWP platform.